docs/content/preview/manage/backup-restore/instant-db-cloning.md

```diff
@@ -0,0 +1,292 @@
+---
+title: Instant database cloning
+headerTitle: Instant database cloning
+linkTitle: Instant database cloning
+description: Clone your database in YugabyteDB for data recovery, development, and testing.
+badges: tp
+menu:
+  preview:
+    identifier: instant-db-clone
+    parent: backup-restore
+    weight: 706
+type: docs
+---
+
+Instant database cloning in YugabyteDB allows you to quickly create a zero-copy, independent writable clone of your database that can be used for data recovery, development, and testing. Cloning is both fast and efficient because when initially created, it shares the same data files with the original database. Subsequently, as data is written to the clone, the clone stores its own changes as separate and independent delta files. Although they physically share some files, the two databases are logically isolated, which means you can freely play with the clone database, perform DDLs, read and write data, and delete it without affecting the original database.
+
+You can create clones as of now, or as of any time in the recent past, within a configurable history retention period. This is particularly useful for data recovery due to user or application error scenarios. For instance, if you dropped a table by mistake at 9:01, then detected this error at 10.45, you are interested in getting the lost data as it was at 9:00 (just before the drop table). At the same time, you don't want to lose any new data added to other tables between 9:01 and 10:45. With databse cloning, you can create a clone of the database at 9:00 (before the table drop) and restore the lost data back to the production database.
+
+![Database clone](/images/manage/backup-restore/db-clone.png)
+
+Cloning has two main use cases:
+
+- Development and testing. Because the two databases are completely isolated, you can experiment with the cloned database, perform DDL operations, read and write data, and delete the clone without impacting the original. Developers can test their changes on an identical copy of the production database without affecting its performance.
+
+- Data recovery. To recover from data loss due to user error (for example, accidentally dropping a table) or application error (for example, updating rows with corrupted data), you can create a clone of your production database from a point in time when the database was in a good state. This allows you to perform forensic analysis, export the lost or corrupted data from the clone, and import it back to the original database.
```

Comment:
The ordering here is a bit weird, since we've already gone through a full example of the data recovery. Can we just say "data recovery, as described in the example above" or re-order this above that point?